### PR TITLE
Add Ruckus vSZ (and SCG) support

### PR DESCRIPTION
**Pull request by eimann**
Hi,

this is to add Ruckus vSZ support and possibly SmartCellGateway support.
Ruckus virtual SmartZone is basically a SmartCellGateway as a VM. It's not a virtualized SmartZone (OS: ruckuswireless-sz). This PR is missing snmpsim etc. tests.

Another issue is the client count per SSID:
- As multiple WLAN Groups can have the same SSID, there's no way of differentiation at the moment
- It would be good to have Domain\Zone\WlanGroup\SSID in the table/graph
I still need to find a good way to do that.

Feel free to work on that and test against SCG as well.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/ruckuswireless-vsz.inc.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Copyright (c) 2015 Gear Consulting Pty Ltd <github@libertysys.com.au>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$hardware      = snmp_getnext($device, '.1.3.6.1.4.1.25053.1.8.1.1.1.1.1.1.3', "-OQv");
```
Comment (murrant) [inline includes/polling/os/ruckuswireless-vsz.inc.php]: You should probably reduce the amount of snmpgetnexts you have here by using snmp_getnext_multi.  (each call takes a decent amount of time)

**Comment by laf** [comment]
@eimann Any update on this?

**Comment by eimann** [comment]
Unfortunately I'm not skilled enough to generate proper snmprec files and I can't give it to a coworker at the moment to fix the missing things. I guess this should be closed for now.

**Comment by murrant** [comment]
@eimann please ask on Discord, it may take a bit but someone will help.

To get the test data, you need a LibreNMS instance that has the device added.
Apply this PR: `./scripts/github-apply 10205`
Then, create the snmprec `./scripts/collect-snmp-data.php -h <device-id/hostname>`
Then add the snmprec to this PR (either commit it to your branch (add-ruckus-vsz) or even just attach it and we'll do that ;)

**Comment by laf** [comment]
Closing this down for now. @eimann if you can get us test data we can at least update your code.

**Pull request closed without merging**